README.md

```diff
@@ -168,7 +168,7 @@ You are free to write your own, which requires you to implement the following in
 public interface Selector<A> {
 
     @Nullable
-    Binding<A> select(final ClientHttpResponse response, final Collection<Binding<A>> bindings) throws IOException;
+    Binding<A> select(final ClientHttpResponse response, final Map<A, Binding<A>> bindings) throws IOException;
```

Comment:
Why does this have to be a map? I explicitly changed this to a Collection because it may not always make sense to think of the bindings as a map.